docs/source/pillars/information_governance.md

```diff
@@ -13,207 +13,599 @@ SATRE Pillars Capability Map
 
 <!-- See all pillars of the SATRE Pillars Capability Map here: {ref}`satre_pillars` -->
 
-## Policy regulation and management
+## Requirements
 
-_How a TRE operator determines what policies and regulations are required and ensures alignment to changes in requirements._
+Information governance requirements are defined by top management within the TRE implementing organisation.
+These will be draw from the context of the organisation, the work it performs and the nature of the data it processes.
+Gathering and monitoring these requirements are a process for the capability along with the overall ownership of risk, definition of controls for those risks and the allocation of resources across the TRE organisation.
+Quality data is a key input to measuring and improving the effectiveness of the TRE in relation to governance, risk and compliance.
+
+These requirements will be different for every TRE organisation.
+TRE organisations should conduct a requirements gathering exercise to identity the applicability of each item in the specification, and to identity the best way to implement them.
```

Comment:
Maybe this paragraph should be in a pre-amble since it applies to all requirements for all pillars?